spec/support/schemas/claims_api/v2/errors/422.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I tested your branch with the following schema  and it worked: schema JSON.parse(File.read(Rails.root.join('spec', 'support', 'schemas', 'claims_api', 'errors',
                                                      'default_with_source.json')))
 so, as Alex said, we don't need the 422.json